### PR TITLE
Fixed an error in ComputeLinearElasticPFFractureStress

### LINKED ISSUE #15695: Fix a bug in ComputeLinearElasticPFFractureStress
## Bug Description

The driving energy term is not correct for the decomposition_type = none case. 

## Steps to Reproduce

The results of the case decomposition_type=none will not give the correct answers. 

## Impact

Fix the tests.

### PR DESCRIPTION
**Pull request by jiangwen84**
close #15695

### CODE REVIEW

**Comment by jiangwen84** [comment]
@hugary1995  Could you take a look at this PR? We have discussed this while ago but I forgot to submit a PR to fix it.

File: modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C [hunk 1]
```diff
@@ -211,14 +211,14 @@ ComputeLinearElasticPFFractureStress::computeQpStress()
       break;
     default:
     {
-      _stress[_qp] = _D[_qp] * _elasticity_tensor[_qp] * _mechanical_strain[_qp] -
-                     _pressure[_qp] * I2 * _I[_qp];
-      F_pos = (_stress[_qp]).doubleContraction(_mechanical_strain[_qp]) / 2.0;
+      RankTwoTensor stress = _elasticity_tensor[_qp] * _mechanical_strain[_qp];
```
Comment (hugary1995) [inline modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C:214]: No pressure now?

Comment (jiangwen84) [reply modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C:214]: good catch! fixed it.

**Review by hugary1995 — COMMENTED** [review]
Looks good to me now.

**Review by fdkong — APPROVED** [review]

**Pull request merged**